Makefile

```diff
@@ -1,4 +1,5 @@
 PROJECTNAME=$(shell basename "$(PWD)")
+VERSION=-ldflags="-X main.Version=$(shell git describe --tags --abbrev=0)"
```

Comment:
If you recall from [the issue](https://github.com/ChainSafe/ChainBridge/issues/533), ideally we want to show tags for releases, but if are not on a tag we should show the commit info. The reason for this is that we want to be able to tell if a binary is built from a specific tag or just some commit. 

I think if we remove `--abbrev=0` we will get the tag if the commit is tagged, otherwise we get the latest tag with the commit info (eg. `v1.1.0-12-ge68ad34`)

Reply:
--abbrev=0 removed